docs/api/endorsement-provisioning/README.md

```diff
@@ -0,0 +1,104 @@
+
+## Provisioning API
+
+Provisioning API provides a facility to provision either Reference Values or Endorsed Values, using a 
+single CoRIM or a batch of CoRIM(s) in a single payload.
+
+* A client who intends to submit a CoRIM or a batch of CoRIM invokes a `POST` on a submit path;
+* If the transaction is complete then a `(200)`response is returned to the client. This is an indication
+  that the submitted CoRIM or batch has been successfully provisioned in a verifier. The ongoing transaction is now complete.
+* For large submission with multiple CoRIMs, the provisioning processing may happen in background. In this case,
+  Client is returned a `(202)` response, with a status of `processing`. Also 
+  client is returned with a session handle to subsequently poll the request for only a given amount of time.
+* The handle has a defined `time to live` upon its expiry automatic garbage collection is performed.
+* The session progresses as (`processing` -> `complete` | `failed`)
+* Client can continue periodic polling of the session uri, until the processing is either `complete` or `failed`
+
+
+## Synchronous submission
+
+```
+ o        (1) POST           .-------------.
+/|\ ------------------------>| /submit     |
+/ \ \ <----------------------|             |
+ \ \ \      200 (OK)         '-------------' 
+
+```
+
+## Asynchronous submission
+
+```
+ o        (1) POST           .-------------.
+/|\ ------------------------>| /submit     |
+/ \ \ <----------------------|             |
+ \ \ \      201 ( Session )  '-------------' 
+  \ \ \                             |
+   \ \ \                            V
+    \ \ \  (2) GET            .-------------.
+     \ \ '------------------->| /session/01 |
+      \ \<--------------------|             |
+             200 (OK)         '-------------'
+             status= "completed"
+```
+
+### Synchronous POST
+
+* Client submits the endorsement provisioning request
+* Server responds with response code `200` indicating successful submission. 
+  The transation is now complete
+```
+>> Request:
+  POST /endorsement-provisioning/v1/submit/?payload=56789abcdesg
+  Host: veraison.example
+  Accept: application/rim+cbor
+
+<< Response:
+  HTTP/1.1 200 OK
+  Content-Type: application/json
+```
+
+### Asynchronous POST
+
+* Client submits the endorsement provisioning request
+* Server responds with response code `201` indicating that the request will be processed asynchronously.
+* Server returns a time bound resource in response header. The resource will be used to poll the original   request till the time it is complete (successful/failed) or the session expires.
+
+```
+>> Request:
+  POST /endorsement-provisioning/v1/submit/?payload=489765abcdesgfdwdunisdufbiubwjndisjd
+  Host: veraison.example
+  Accept: application/rim+cbor
+
+<< Response:
+  HTTP/1.1 201 Created
+  Content-Type: application/session-response+json
+  Location: https://veraison.example/endorsement-provisioning/v1/session/1234567890
```

Comment:
Location can be a relative URL, it's not necessary to make it absolute.

Reply:
I agree, have corrected it now!